contracts/tgrade-validator-voting/src/msg.rs

```diff
@@ -101,6 +100,16 @@ pub enum ValidatorProposal {
         /// The contract address to be cleared
         contract: String,
     },
+    ClientUpdate {
+        /// The client identifier for the client to be updated if the proposal passes
+        subject_client_id: String,
+        /// The substitute client identifier for the client standing in for the subject client
+        substitute_client_id: String,
+    },
+    Upgrade {
```

Comment:
And here too, a proper `Ibc` prefix to differentiate this update proposal from the chain upgrade one.

Reply:
```suggestion
    IBCUpgrade {
```

Reply:
Thanks for the comments.

As I understand it, this message should go entirely, as it's a duplicate of the `RegisterUpgrade` message above. As such, i's also not related to IBC.

Then, in the `GovProposal` enum (`bindings/src/gov.rs`) , both `ClientUpdate` and `Upgrade` should go, and the current `IBCClientUpdate` message should be properly updated.